src/canvas-view/canvas-editor/toolbox-items.component.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('toolboxItems', {
+      templateUrl: 'canvas-view/canvas-editor/toolbox-items.html',
+      bindings: {
+        items: '=',
+        startDragCallback: '=',
+        clickCallback: '=',
+        searchText: '='
+      },
+      controller: function toolboxItemsController () {
+        var ctrl = this;
+
+        ctrl.clickCallbackfmDir = function (item) {
+          if (!item.disableInToolbox) {
+            ctrl.clickCallback(item);
+          }
+        };
+
+        ctrl.startDragCallbackfmDir = function (event, ui, item) {
+          ctrl.startDragCallback(event, ui, item);
```

Comment:
Same call out on function check...
```js
if (angular.isFunction(ctrl.startDragCallback)) {...
```

Reply:
Agree..

Reply:
Same as above ```ctrl.startDragCallback``` isn't a function either. Just being used for binding and to pass the arguments into from the it's parent function.